### ISSUE TITLE
[Feature Request] Refresh Dashboard

### ISSUE THREAD

**Issue by cesalo** [gotson/komga#202]
### Is your feature request related to a problem? Please describe.
This is not critical or anything like that but thought would be nice to have it.
When I upload some comics and I'm in the dashboard I need to go to another link and go back to Home to see the new comics/series or refresh my browser.
I'm using a tablet for reading comics and have Komga with a shortcut in full screen so don't have the tab or url to refresh the whole browser. I guess for other ppl just clicking on F5 or refresh the tab is easy enough - again not critical so feel free to dismiss this :).
### Describe the solution you'd like
- That clicking in Home when being in Home (Dashboard) actually updates the dashboard, at the moment it seems it doesn't do anything therefore the need to go to another link and go back to Home.
- Another option is that there is an icon visible all the time that would refresh the Dashboard when clicked on it:
![Screenshot from 2020-06-10 23-35-19](https://user-images.githubusercontent.com/16859916/84342254-1c250b80-ab73-11ea-88a9-f9adfaab5558.png)
- Another option (I assume much more complicated) that Komga pushes the status change and automatically refreshes the dashboard (like in Plex when there is a library scan).

**Comment by gotson** [comment]
The first option doesn't work, because duplicated navigation are not allowed by Vue Router.

I'm not a fan of the second option.

The third is the best indeed, and will depend on #124, which is exactly the kind of use case for that other feature.
